js/import/rules.import.js

```diff
@@ -0,0 +1,100 @@
+
+import xPathToCss from '../libs/vendors/xpath-to-css/xpath-to-css.js';
+
+const XPATH_BODY = '/html[1]/body[1]';
+
+const baseTransformRules = {
+  cleanup: {
+    start: [],
+    end: [],
+  },
+  blocks: [
+    {
+      type: 'metadata',
+      insertMode: 'append',
+      params: {
+        metadata: {}
+      },
+    },
+  ]
+};
+
+/*
+function selectElementFromXpath(xpath, document) {
+  return document.evaluate(xpath, document, null, XPathResult.FIRST_ORDERED_NODE_TYPE, null).singleNodeValue;
+}
```

Comment:
Is `utils.js` available here?  It already has this code with some error catching.